pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/TemplateActionType.java

```diff
@@ -0,0 +1,7 @@
+package org.broadinstitute.ddp.studybuilder.task;
+
+public enum TemplateActionType {
+    ADD,
```

Comment:
Maybe remove `ADD` and `DELETE` but leave a note here saying that these might be things we want to build out later.